### DISCUSSION TITLE
이미지 집합 차례로 바꿔서 비디오처럼 재생하기

### DISCUSSION THREAD

**Discussion by JeongDongBaek** [naverz/zepeto-studio-kor#209] [Issue Report (Legacy)]
Resoruces 폴더로부터 불러온 images 집합들을 이용하여 차례로 불러 적용하여 비디오처럼 보이게 하는 코드입니다.
유니티 에디터 (with Zepeto Server)에서는 잘 작동하는 것을 볼 수 있으나, QR를 통한 모바일에서는 0번째 이미지에서 작동이 되지 않습니다.
무엇이 문제일까요?

적용은 transprent 재질의 박스로부터 불러내어 mainTexture를 바꾸는 형식으로 진행하고 있습니다.

https://user-images.githubusercontent.com/33086479/149309347-d1cb8160-9210-415a-b695-10a3d0263270.mp4

![image](https://user-images.githubusercontent.com/33086479/149304846-7434d046-10e8-494e-82b9-3b5d0cff41ee.png)

실험 결과

옛날 패키지 버전으로 downgrade해도 안됩니다.
패키지 최신버전으로 upgrade해도 안됩니다.
sharedMaterial (Meterial as Instance) 둘다 안됩니다.
Mat를 transparent -> opaque 바껴도 안됩니다
캐싱 위치 바꿔도 안됩니다.
Update문으로 빼도 안됩니다.

**Comment by hyeonjin-jo** [comment]
안녕하세요.

올려주신 코드로 테스트 결과 모바일환경에서도 정상동작하는 것을 확인했습니다.

this.path_pics 변수에 대한 조건문이 존재하는데 이부분에 문제가 없는지 혹은 다른 부분에서 발생한 오류가 아닌지 체크가 필요할 것 같습니다.
